### DISCUSSION TITLE
Feature Request: Memories, See photo's from this day years ago

### DISCUSSION THREAD

**Discussion by JackJones2018** [photoprism/photoprism#718] [General]
A memories feature like facebook, on this day x amount of years ago these are your memories? Piwigo has a feature like this. Face detection is moving forward on photoprism quicker than piwigo i think. Synology moments does face detection but seemed laggy when i used it last and it needed its own folder order i believe.
thanks

**Comment by graciousgrey** [comment]
I like this idea. We should convert this to an issue :) I just do not find the functionality

**Reply by graciousgrey** [reply]
I created this as an issue. https://github.com/photoprism/photoprism/issues/720. Feel free to give it a 👍
